validator/client/src/main/java/tech/pegasys/teku/validator/client/BeaconProposerPreparer.java

```diff
@@ -55,12 +57,14 @@ public class BeaconProposerPreparer implements ValidatorTimingChannel, FeeRecipi
       ValidatorIndexProvider validatorIndexProvider,
       ProposerConfigProvider proposerConfigProvider,
       Optional<Eth1Address> defaultFeeRecipient,
+      UInt64 defaultGasLimit,
```

Comment:
variables should all really be 'final'.. in function definition and when declared, we may as well fix this file while we're here... The only time we don't use final is in an `interface` as a rule.